lib/api.js

```diff
@@ -33,8 +33,7 @@ module.exports = settings => {
 
   flow.migrate();
 
-  app.get('/', excludeAutoResolved);
-
+  app.use('/tasks', tasksRouter(flow));
```

Comment:
I added this route to provide a way to hit the standard taskflow `/` route if we still want it - do we still want it? If not I'll just make this `/`.

Reply:
I don't think there's a good reason for us to want to go to the underlying '/' route, so I'd mount this on the root.